docs/relnotes.md

```diff
@@ -6,6 +6,8 @@ hide:
 # Release Notes
 
 This is the list of changes to Taipy releases as they were published.
+Please refer to [upgrades](./legacy-version-migration.md) page for
```

Comment:
I would not use 'upgrade' here, which would be used when moving from any X.Y to Z.T (Z.T > X.Y).
"migration" makes more sense to me.